nowcasting_metrics/metrics/mae.py

```diff
@@ -401,28 +437,36 @@ def make_mae(
         if model not in max_forecast_horizon_minutes:
             max_forecast_horizon_minutes[model] = 480
 
-    # national
+    # 1. Calculate the MAE for the forecast values for each model
     for model_name in models:
-        make_mae_one_gsp(
-            session=session, datetime_interval=datetime_interval, gsp_id=0, model_name=model_name
-        )
+
+        if model_name not in all_forecast_values:
+            logger.warning(f"No forecast values for model {model_name} for me, skipping...")
+            continue
+
+        forecast_values_df = all_forecast_values[model_name]
 
         # loop over forecast horizons
-        for forecast_horizon_minutes in get_forecast_range(max_forecast_horizon_minutes[model_name]):
-            make_mae_one_gsp_with_forecast_horizon(
+        for forecast_horizon_minutes in [None] + list(get_forecast_range(max_forecast_horizon_minutes[model_name])):
```

Comment:
Why always have the [None] entry at the first element?

Reply:
What we did before is run one metric with no forecast_horizon_minutes hence None, + the list. 
Perhaps I could add comment to help explain this